README.md

```diff
@@ -154,24 +154,8 @@ Running tests locally is convenient, but before your changes can be
 merged, we need to test them on all supported distributions and
 architectures.
 
-When you submit patches to gerrit, oVirt's Jenkins CI will run its tests
-according to configuration in the stdci.yaml file.
-
-### Travis CI for storage patches
-
-oVirt's Jenkins CI is the integrated method for testing Vdsm patches,
-however for storage related patches we have to cover also 4k tests which
-are not covered currently by Jenkins CI. This can be achieved in a fast
-way manually and independently from gerrit by invoking Travis CI on your
-github branch:
-
-- Fork the project on github.
-- Visit https://travis-ci.org, register using your github account, and
-  enable builds for your Vdsm fork.
-- Push your changes to your github fork to trigger a build.
-
-See .travis.yml file for tested Travis platforms and tests configurations.
-
+When you push patches to GitHub, CI will run its tests according to the
+configuration in the .github/workflows/ci.yml file.
```

Comment:
Uhm, first to clarify: I think I misread, you said it would be better in a different patch (before this one), and I just thought you meant a different commit previous to the split. So sorry about that, I can still revert it and split it if you think it is better that way.

And about removing the section: you wrote that Travis CI was not used for a while, and that we could drop the section. I wanted not just to drop it, but try to update it, and found the GitHub CI config file.

I have documented the change in the commit message, but again, I could drop the change and do it in a separate patch as you initially pointed.

Reply:
I commented about the missing commit message. I know why you removed this but the person reading git log in 6 month will not have any idea if you don't explain this in the commit message.